src/CoreFx.Private.TestUtilities/src/System/Diagnostics/RemoteExecutorTestBase.cs

```diff
@@ -199,5 +209,7 @@ public sealed class RemoteInvokeOptions
 
         public int TimeOut {get; set; } = RemoteExecutorTestBase.FailWaitTimeoutMilliseconds;
         public int ExpectedExitCode { get; set; } = RemoteExecutorTestBase.SuccessExitCode;
+        public string ExceptionFile { get; } = Path.GetRandomFileName();
```

Comment:
Should we instead create this file in the temp path? I don't know if the fact that this is a remote process you're allowed to create a file in the common path when running inside app container, I would say that it should let you but I'm not certain as creating files outside the app container is always a security issue.